common/utils/src/main/resources/error/error-conditions.json

```
@@ -5309,6 +5309,11 @@
           "Subquery expressions are not supported within higher-order functions. Please remove all subquery expressions from higher-order functions and then try the query again."
         ]
       },
+      "INLINE_TABLE_CONTAINS_SCALAR_SUBQUERY" : {
```

Let's make it shorter since it could appear in users docs:
```suggestion
      "SCALAR_SUBQUERY_IN_VALUES" : {
```